extension.html

```diff
@@ -55,13 +55,13 @@ <h2>Introduction</h2>
 	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs
         and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events
         that were coalesced into a single event.
-        Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually.
+        User agents may coalesce a stream of events into a single event sent to javascript.
+        This situation can occur when many events are generated by the hardware in between the requestAnimationFrame boundaries or for long running javascript code that misses the frame boundary.
         Basically if the javascript handlers process the events and update the screen contents faster than the screen refresh rate only the last update will be rendered and shown in the next frame.
```

Comment:
Basically, if the javascript handlers process the events and update the screen contents faster than the screen refresh rate only the last update will render and show in the next frame.